rollup.config.js

```diff
@@ -4,9 +4,18 @@ import external from 'rollup-plugin-peer-deps-external'
 import commonjs from '@rollup/plugin-commonjs'
 import { visualizer } from 'rollup-plugin-visualizer'
 import json from '@rollup/plugin-json'
-import packageJson from './package.json' assert { type: 'json' }
```

Comment:
To avoid having compatibility issues with other Node versions using `with` instead of `assert`, the solution above was added ⬇️

Reply:
Know more here: https://stackoverflow.com/questions/78876691/syntaxerror-unexpected-identifier-assert-on-json-import-in-node-v22